src/components/Questions/Header.vue

```diff
@@ -7,6 +7,14 @@
       @click="togglePalette"
       data-test="togglePaletteButton"
     ></icon-button>
+    <div class="flex space-x-3">
+      <!-- timer button / can't click -->
```

Comment:
also rename it to `timerCountdown` or something instead of button?
It's not really a button so that's why even though the component we're using is an icon button

Reply:
yeah.. changed it to `countdownTimer`